pyoxidizer/src/py_packaging/distribution.rs

```diff
@@ -331,6 +351,101 @@ pub enum ExtensionModuleFilter {
     NoGPL,
 }
 
+pub struct PythonPaths {
+    pub prefix: PathBuf,
+    pub bin_dir: PathBuf,
+    pub python_exe: PathBuf,
+    pub stdlib: PathBuf,
+    pub site_packages: PathBuf,
+    pub pyoxidizer_state_dir: PathBuf,
+}
+
+/// Resolve the location of Python modules given a base install path.
+pub fn resolve_python_paths(base: &Path, python_version: &str) -> PythonPaths {
```

Comment:
I suspect that this should be converted into a more normal class with methods which more clearly handle the logic.
Currently there is a bit of voodoo in here for handling `virtualenv` on Windows which puts things in yet-another-layout.  Removing that will make this less brittle/confusing.